pom.xml

```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.github.kislayverma</groupId>
     <artifactId>rulette</artifactId>
-    <version>1.2.7-SNAPSHOT</version>
+    <version>1.2.8-SNAPSHOT</version>
```

Comment:
No need to change version. Master is already at 1.2.7-SNAPSHOT and that is the next milestone. 1.2.8 will be the entirely nxt iteration, after 1.2.7 is released.